src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.File;
+
+public class Main {
+    public static void main(String[] args) {
+        String [] files = new String[] {
+                "apple.csv",
+                "banana.csv",
+                "grape.csv",
+                "orange.csv"
+        };
+        String [] resultFiles = new String[]{
+                "apple_result.txt",
+                "banana_result.txt",
+                "grape_result.txt",
+                "orange_result.txt"
+        };
+
+        WorkWithFile workWithFile = new WorkWithFile();
+        for (int i = 0; i < files.length; i++) {
+            workWithFile.getStatistic(files[i], resultFiles[i]);
+        }
+    }
+}
```

Comment:
The Main class should be the entry point of the program and should not contain additional logic such as arrays of file names. Consider moving these arrays and the loop that processes them into a separate method or class that handles file operations.